app/js/components/common/Header.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { Link, IndexLink } from 'react-router';
+
+const NUMBER_OF_COLUMNS = 3;
+
+// Dropdown data to be displayed
+const menu = [
+  'Inpatient ward',
+  'Isolation ward',
+  'Laboratory',
```

Comment:
Yes I did hard code them for now, cos @andela-odaniel and I agreed on working on the UI first and when we're done with that we can begin making a call to the APIs. So I mocked the data for now. @dkayiwa